### ISSUE TITLE
Possibly put ABI json files in tasit-contracts

### ISSUE THREAD

**Issue by pcowgill** [tasitlabs/TasitSDK#144]
If not, have both the ABIs in tasit-action in the same subdirectory

**Comment by marcelomorgado** [comment]
Note: I've put only ERC721Full ABI inside `abi/` dir because it's the only type of contract supported for de SDK by now. And kept SimpleStorage JSON inside `testHelpers/` because it's using only for tests.

**Comment by pcowgill** [comment]
I could see an argument either that `ERC721Full` is a sample contract that we want to feature as supported by the Tasit SDK or that people might want to use for developing other projects, in which case it makes to have it available as a modular package in `tasit-contracts`, or that it's used primarily for testing just the `tasit-action` package and belongs alongside `SimpleStorage`

**Comment by marcelomorgado** [comment]
I think that are `3` places where contracts abi can live:

> is a sample contract that we want to feature as supported by the Tasit SDK 

Encapsulated by `NFT` class and inside of the `abi/` dir. (manually)

> that people might want to use for developing other projects, in which case it makes to have it available as a modular package in tasit-contracts, 

Inside `tasit-contracts` `build/` dir. (automatically)

> that it's used primarily for testing just the tasit-action package and belongs alongside SimpleStorage

Inside `tasit-action` `testHelpers/` dir. (manually for now https://github.com/tasitlabs/TasitSDK/issues/45)

**Comment by pcowgill** [comment]
Hmm okay, that makes sense to me! Thanks.

I think the only thing I would change is the name of the `abi` dir so that what type of contracts live there is immediately clear.

And maybe if there will be some non-abi test helpers in `tasit-action` `testHelpers`, then we'd eventually create a subdir for just the contracts ABIs in that folder. But it would be putting the cart before the horse to do that now.

**Comment by pcowgill** [comment]
Our policy on this has changed since we last discussed it in this issue, right?

**Comment by marcelomorgado** [comment]
Yes. Now, all packages are importing contracts ABIs from `tasit-contract/build/contracts`, the one exception is ABI that is been used by "NFT without ABI" feature (`tasit-action/src/abi/ERC721Full.json`).
